main/electron.ts

```diff
@@ -46,11 +46,11 @@ export function createWindow(): void {
   mainWindow.on('closed', (): void => {
     mainWindow = undefined;
   });
-  mainWindow.webContents.on('new-window', (event: Event, url: string): void => {
-    if (url.startsWith('http:') || url.startsWith('https:')) {
-      event.preventDefault();
-      shell.openExternal(url);
+  mainWindow.webContents.setWindowOpenHandler((details: HandlerDetails) => {
+    if (details.url.startsWith('http:') || details.url.startsWith('https:')) {
+      shell.openExternal(details.url);
     }
+    return { action: 'deny' };
```

Comment:
This part doesn't seem to be working correctly. A command prompt window opens briefly when starting and finishing rendering and when closing the program after rendering. I found this issue talking about it:
https://github.com/electron/electron/issues/27967

Reply:
It wasn't `app.openPath` that was causing the console windows but the calls to spawn Ffmpeg.
I've fixed everything apart from the one when closing the app after a render.